client_test/virtual_fund_message_delay_test.go

```diff
@@ -0,0 +1,64 @@
+package client_test
+
+import (
+	"math/big"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	"github.com/statechannels/go-nitro/client/engine/messageservice"
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/protocols/virtualfund"
+	"github.com/statechannels/go-nitro/types"
+)
+
+const MAX_MESSAGE_DELAY = time.Millisecond * 100
+
+// Since we are delaying messages we allow for slightly more time to complete the objective
+const OBJECTIVE_TIMEOUT = time.Second * 2
```

Comment:
Nit 1: I would replace "slightly more time" with "enough time" in the comment (since we don't know what it is slightly more than...)


Nit 2: Consider putting these consts  _inside_ `TestVirtualFundWithMessageDelays`, unless you want to use them in other tests in the `client_test` package.